project-base/src/Controller/Front/RobotsController.php

```diff
@@ -5,47 +5,36 @@
 namespace App\Controller\Front;
 
 use Shopsys\FrameworkBundle\Component\Domain\Domain;
+use Shopsys\FrameworkBundle\Model\Seo\SeoSettingFacade;
 use Shopsys\FrameworkBundle\Model\Sitemap\SitemapFilePrefixer;
 use Symfony\Component\HttpFoundation\Response;
 
 class RobotsController extends FrontBaseController
 {
-    /**
-     * @var \Shopsys\FrameworkBundle\Component\Domain\Domain
-     */
-    private $domain;
-
-    /**
-     * @var \Shopsys\FrameworkBundle\Model\Sitemap\SitemapFilePrefixer
-     */
-    private $sitemapFilePrefixer;
-
-    /**
-     * @var string
-     */
-    private $sitemapsUrlPrefix;
-
     /**
      * @param string $sitemapsUrlPrefix
      * @param \Shopsys\FrameworkBundle\Component\Domain\Domain $domain
      * @param \Shopsys\FrameworkBundle\Model\Sitemap\SitemapFilePrefixer $sitemapFilePrefixer
+     * @param \Shopsys\FrameworkBundle\Model\Seo\SeoSettingFacade $seoSettingFacade
      */
     public function __construct(
-        string $sitemapsUrlPrefix,
-        Domain $domain,
-        SitemapFilePrefixer $sitemapFilePrefixer
+        protected readonly string $sitemapsUrlPrefix,
+        protected readonly Domain $domain,
+        protected readonly SitemapFilePrefixer $sitemapFilePrefixer,
+        protected readonly SeoSettingFacade $seoSettingFacade,
```

Comment:
This is considered a BC-break, you need to make this parameter nullable and add dependency via @required method - see usages of SetterInjectionTrait::setDependency()